.env.example

```
@@ -1,40 +1,10 @@
-# Grafana
-GRAFANA_API_KEY=your-grafana-api-key
-GRAFANA_PASSWORD="VaalEmpire2025!"
-
-# Vercel Deployment
-VERCEL_TOKEN=your-vercel-token
-VERCEL_ORG_ID=your-org-id
-VERCEL_PROJECT_ID=your-project-id
-HUBSPOT_PRIVATE_APP_TOKEN='your-hubspot-private-app-token'
+# App Config
+NEXT_PUBLIC_API_ENDPOINT=https://your-api.com
+NODE_ENV=development
```

dotenv-linter (4.0.0)

[warning] [EndingBlankLine] No blank line at the end of the file

(EndingBlankLine)